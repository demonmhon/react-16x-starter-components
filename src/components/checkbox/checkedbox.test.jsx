import React from 'react';
import { create } from 'react-test-renderer';
import Checkbox from './index';

describe('Checkbox component', () => {
  test('it matches the snapshot', () => {
    const component = create(<Checkbox />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
