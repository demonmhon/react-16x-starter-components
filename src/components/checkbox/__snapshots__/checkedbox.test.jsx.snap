// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox component it matches the snapshot 1`] = `
<label
  className="checkbox"
>
  <div
    className="checkbox__container"
  >
    <input
      checked={false}
      onChange={[Function]}
      type="checkbox"
    />
    <span
      className="checkbox__ui"
    />
    <div
      className="checkbox__label"
    >
      
    </div>
  </div>
</label>
`;
